modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -30,17 +30,16 @@ You can deploy {lvms} using one of the following:
 [id="lvms-deployment-requirements-for-sno-ran_{context}"]
 == Requirements
 
-Before deploying {lvms} on {sno} clusters, ensure that the following requirements are met:
+Before deploying {lvms}, ensure that every managed cluster has dedicated disks that are used to provision storage.
 
-* You have installed {rh-rhacm-first} on an {product-title} cluster.
-* Every managed cluster has dedicated disks that are used to provision storage.
-
-Before deploying {lvms} in a private CI environment where you can reuse the virtual machines and storage devices, ensure that you have wiped the disks that are not in use. 
+Before deploying {lvms} in a private CI environment where you can reuse the storage devices that you configured in the previous {lvms} installation, ensure that you have wiped the disks that are not in use. If you do not wipe the disks before installing {lvms}, you cannot reuse the disks without manual intervention.
 [NOTE]
 ====
 You cannot wipe the disks that are in use.
 ====
 
+If you want to install {lvms} by using {rh-rhacm-first}, ensure that you have installed {rh-rhacm} on an {product-title} cluster.
+
```

Comment:
This plus is hidden, luckily, but sometimes it can rear it's nasty head in the published docs. I think it can be removed without causing any issues as no indentation is required for this intro block.

Reply:
Updated as per the suggestion.